package.json

```diff
@@ -15,31 +15,26 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "peerDependencies": {
-    "apollo-cache-inmemory": "^1.1.5",
-    "apollo-client": "^2.2.0",
-    "apollo-link": "^1.0.7",
-    "apollo-link-http": "^1.3.2",
-    "bs-platform": "^2.1.0",
-    "graphql-tag": "^2.6.1",
+    "apollo-cache-inmemory": "^1.2.1",
+    "apollo-client": "^2.3.1",
+    "apollo-link": "^1.2.2",
+    "apollo-link-http": "^1.5.4",
+    "bs-platform": "^3.1.3",
+    "graphql-tag": "^2.9.2",
     "graphql_ppx": "^0.2.0",
-    "react-apollo": "^2.1.3"
+    "react-apollo": "^2.1.4"
```

Comment:
Unrelated to this PR, but I think we need to find a more sustainable approach of dealing with all those "peer" dependencies. I'll probably open up a new issue to discuss this.